packages/algob/src/internal/deployer.ts

```diff
@@ -498,17 +500,17 @@ class DeployerBasicMode {
   }
 
   /**
-   * Asserts that there is a checkpoint with given lsig name.
-   * @param lsigName name of the lsig (passed by user during mkContractLsig/mkDelegatedLsig)
+   * Throws error if application info is not present in CP
+   * @param key key against which app information is stored in checkpoint
    */
-  assertLsigExistsInCP (lsigName: string): LogicSigAccount {
-    const lsig = this.getLsigByName(lsigName);
-    if (lsig === undefined) {
-      throw new BuilderError(ERRORS.GENERAL.LSIG_NOT_FOUND_IN_CP, {
-        lsigName: lsigName
+  assertAppExistsInCP (key: string): rtypes.AppInfo {
```

Comment:
need to add this change to changelog

Reply:
this is used internally, do we need to add this changelog? Maybe we could make in `private`?